Signal/src/view controllers/CallViewController.swift

```diff
@@ -519,7 +519,6 @@ class CallViewController: UIViewController, CallDelegate {
 
     func updateCallUI(callState: CallState) {
         let textForState = localizedTextForCallState(callState)
-        Logger.info("\(TAG) new call status: \(callState) aka \"\(textForState)\"")
```

Comment:
Whoops. I didn't intend to delete the logging line, I'll put it back, but the substantive part of the commit was to *only* redraw the status label portion of the view 20 x per second, rather than the rest of the updateCallUI method.